### PR TITLE
Updated the Readme

### PR DESCRIPTION
**Pull request by ghost**
Made things a bit less shouty and a bit more helpy!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,33 @@
...
+3. To use zoom.js, simply add a `data-action="zoom"` attribute to an image element you would like to utilise the plugin. For example:
 
-This plugin ~probably~ works in all relevant browsers. If not, let me know. 
\ No newline at end of file
+	`<img src="img/blog_post_featured.png" data-action="zoom">`
+
+
+### Awesome Features
+
+zoom.js will automatically close when the user clicks out of it, or presses the escape key.
```
Comment (jdreesen) [inline README.md]: Or scrolls.

Comment (ghost) [inline README.md]: Very true! Added :-)

**Comment by lipis** [comment]
I would also break the long lines at 79 chars as it should be also readable on other devices :)

Of course squashing will be necessary in the end to avoid the noise..

**Comment by fat** [comment]
got it, thanks!

**Pull request closed without merging**